platforms/r3-corda/configuration/reset-network.yaml

```diff
@@ -132,4 +154,4 @@
     privilege_escalate: false           #Default to NOT escalate to root privledges
     install_os: "linux"                 #Default to linux OS
     install_arch:  "amd64"              #Default to amd64 architecture
-    bin_install_dir:  "~/bin"            #Default to /bin install directory for binaries
+    bin_install_dir:  "~/bin"            #Default to /bin install directory for binaries
```

Comment:
Always have a new line in files(no extra spaces, nothing, just press enter at the last line)